src/inputs/internal/SelectFieldBase.tsx

```diff
@@ -245,6 +245,7 @@ export function SelectFieldBase<O, V extends Value>(props: SelectFieldBaseProps<
     shouldFlip: true,
     isOpen: state.isOpen,
     onClose: state.close,
+    placement: "bottom",
```

Comment:
`placement: 'bottom'` is the default, but hard coding it now because we make assumptions on this default in the ListBox component.